js/src/features/project/selectors.ts

```diff
@@ -18,5 +17,15 @@ export const getRolePermissions = (state: RootState) =>
 export const getIsOpenCreateTeamForm = (state: RootState) =>
   state.project.project.isOpenCreateTeamForm;
 
-export const getProjectTeam = (state: RootState) =>
-  state.project.team.data;
+export const getProjectTeam = (state: RootState) => state.project.team.data;
+
+export const getIsOpenTeamAddMemberForm = (state: RootState) =>
+  state.project.team.isOpenAddMemberForm;
+
+export const getTeamErrors = (state: RootState) => state.project.team.errors;
+
+export const getEmployees = (state: RootState) =>
```

Comment:
Looks like you return employees, but you return employeesSelectOptions

Reply:
Fixed